assets/css/main.css

```diff
@@ -0,0 +1,160 @@
+/*====================
+  Container
+======================*/
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+.container {
+  /* position: relative; */
+  background-image: url("../images/hero.jpg");
```

Comment:
Your syntax is correct and works on local machine, try below to fix the previewer problem.
- Upload hero.jpg image to a desired platform eg [Cloudinary](cloudinary.com)
- Get Url to hero.jpg from which ever platform you use eg [Url](https://res.cloudinary.com/drw0cxkwk/image/upload/v1544707606/Microverse_css_grid_project/weird.jpg)
- Add url to background-image: url([Url](https://res.cloudinary.com/drw0cxkwk/image/upload/v1544707606/Microverse_css_grid_project/weird.jpg));

Reply:
Done